packages/marshal/src/marshal-justin.js

```diff
@@ -445,3 +451,60 @@ const decodeToJustin = (encoding, shouldIndent = false, slots = []) => {
 };
 harden(decodeToJustin);
 export { decodeToJustin };
+
+/**
+ * @param {Passable} passable
+ * @param {boolean} [shouldIndent]
+ * @returns {string}
+ */
+export const passableAsJustin = (passable, shouldIndent = true) => {
+  let slotCount = 0;
+  // eslint-disable-next-line no-plusplus
+  const convertValToSlot = val => `s${slotCount++}`;
+  const { toCapData } = makeMarshal(convertValToSlot);
+  const { body, slots } = toCapData(passable);
+  const encoded = JSON.parse(body);
+  return decodeToJustin(encoded, shouldIndent, slots);
+};
+harden(passableAsJustin);
+
+// The example below is the `patt1` test case from `qp-on-pattern.test.js`.
+// Please co-maintain the following doc-comment and that test module.
+/**
+ * `qp` for quote passable as a quasi-quoted Justin expression.
```

Comment:
Now that this landed, I'd expect it to appear in the list of function on https://endojs.github.io/endo/modules/_endo_marshal.html . But I don't see it. Anybody know why not?

Reply:
I do not.